firedrake/adjoint/function.py

```diff
@@ -52,7 +52,8 @@ def wrapper(self, *args, **kwargs):
                 output = split(self, *args, **kwargs)
 
             if annotate:
-                output = tuple(firedrake.Function(output[i],
+                output = tuple(firedrake.Function(output[i].function_space(),
+                               output[i],
```

Comment:
Can we fix the indentation here please? I realise that this is a pre-existing issue but this is very confusing. The arguments to the `Function` constructor should be indented to the opening bracket of that constructor, not the opening bracket of the `tuple`.

Reply:
Done!